web/frontend/package.json

```diff
@@ -26,12 +27,14 @@
     "i18next-browser-languagedetector": "^6.1.3",
     "prop-types": "^15.8.1",
     "react": "^17.0.2",
+    "react-beautiful-dnd": "^13.1.0",
```

Comment:
You need to add the corresponding typescript types: https://www.npmjs.com/package/@types/react-beautiful-dnd

Reply:
(as a dev dependency: `npm install -D @types/react-beautiful-dnd`)